packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -461,9 +506,31 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20 {
     }
 
     /**************************************
-     *        INTERNAL FUNCTIONS          *
+     *    INTERNAL & PRIVATE FUNCTIONS    *
      **************************************/
 
+    // Update the internal fee counter metrics by adding in any accumulated fees from the last time this logic was
+    // called. Considers any allocated fees that are assigned to the LPs at the conclusion of a bridging action.
+    function updateFeeCounters(uint256 allocatedLpFees) internal {
```

Comment:
Two questions:
1. Should we make a separate method for _adding new fees_ as opposed to distributing fees? They seem like distinct actions.
2. nit: variable name seems a little confusing. Thoughts on `newFees` or something similar?

Reply:
yup. good points. I split this into two new re-named methods: 1) `updateAccumulatedLpFees()` and `allocateLpFees(uint256 allocatedLpFees)`